schemas/metadata/metaschema/metaschema.1.schema.json

```diff
@@ -0,0 +1,62 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
```

Comment:
I'm not sure that this schema belongs in `schemas/`, since it's defining what is valid in _other_ schemas in `schemas/`. This might belong in a separate `validation/` directory.